recipes/conversation_view/feed_item_row/feed_item_row.stories.js

```diff
@@ -0,0 +1,131 @@
+import { action } from '@storybook/addon-actions';
+import { createTemplateFromVueFile, getIconNames } from '@/common/storybook_utils';
+import DtRecipeFeedItemRow from './feed_item_row';
+import DtRecipeFeedItemRowMdx from './feed_item_row.mdx';
+import DtRecipeFeedItemRowDefaultTemplate from './feed_item_row_default.story.vue';
+import DtRecipeFeedItemRowVariantsTemplate from './feed_item_row_variants.story.vue';
+
+const iconsList = getIconNames();
+
+export const argsData = {
+  avatarImageUrl: 'https://i1.sndcdn.com/avatars-000181324408-652e57-t500x500.jpg',
+  displayName: 'Dwight Schrute',
+  time: '4:54 PM',
+  shortTime: '4:54',
+  onFocus: action('focus'),
+  onHover: action('hover'),
+  default: `<p>Elementum fames nullam elementum velit proin vitae aliquet.
+  Platea nulla consectetur consequat sagittis nullam et ultricies nisl rhoncus
+  aliquet elementum venenatis quisque.</p>`,
+  menu: 'menu',
+};
+
+/*
+  Controls
+  ========
+
+  Here we define any custom controls or control overrides for our components.
+
+  By default storybook will attempt to provide an appropriate control of the same name for each property in the
+  component as well as include any description provided using a prop decorator within your component (see above).
+
+  Storybook will also attempt to provide an appropriate control for each slot in the component as well as include any
+  description provided using a slot decorator within your component (see below).
+
+  <!-- @slot example slot decorator -->
+*/
+export const argTypesData = {
```

Comment:
Just one small thing, add

```
onFocus: {
  table: {
    disable: true,
  },
},
onHover: {
  table: {
    disable: true,
  },
},
```

to get rid of these awkward uncategorized properties in the table

![Screenshot 2023-04-24 at 11 01 12 PM](https://user-images.githubusercontent.com/64808812/234188152-8d5ddbf6-a07e-403f-85a2-a77d3328eab1.png)